test-cases/scale/longevity-elasticity-many-small-tables.yaml

```diff
@@ -0,0 +1,36 @@
+# This is a test case for having 90% disk utilization with a lot of small tables.
+# The data is split equally among 500 tables.
+# The dataset size is aligned with 'i4i.xlarge'.
+# It uses a c-s user-profile template for all 500 tables.
+# It runs 4 batches of 125 tables each.
+# On each batch cycle, 125 tables are created, then a load is generated for all of these tables.
+# When all the 125 stress writes/reads are done, it continue with the next batch until stress to all 500 tables is completed (after 4 cycles).
+# Each one of the 500 tables has both write and read load.
+
+test_duration: 480
+
+cs_duration: '0m'
+cs_user_profiles:
+    - data_dir/templated-elasticity-tables.yaml
+
+pre_create_schema: true
+user_profile_table_count: 501
+batch_size: 125
+
+n_loaders: 5
```

Comment:
why we need 5 big loaders against 3 smaller db nodes?

Reply:
anyway, one loader possibly could handle all of this - 30kops is not that much. If we need 5, then smaller ones would suffice

Reply:
1. it runs 125 per cycle. round robin is not relevant here.
2. It still has a problem that the cycles, from some reason, continues, so in last test in got to more than 600 stress threads since a 5th cycle started.
3. @soyacz , it's not a matter of ops but rather the number of parallel stress threads. When i tried, for example, 250 threads per cycle, the test crashed (probably loader issues like OOM or similar).

Reply:
@fruch , it is explicitly set here to True: https://github.com/scylladb/scylla-cluster-tests/blob/489ec24a2618bf2a02e4a4c482126710b79bc84f/longevity_test.py#L334

Reply:
Not sure this calculation is right.
In this case there is 5 * 16 GB memory = 80GB total memory for the loaders.
But according to above memory requirements: 125 threads * 1.3GB = 162.5GB, so according to above calculation the test should have failed with OOM.
@pehala , i think this consideration might be negligible since this test may only be run once a year or so, so it may spare a cost of ~ 5$ a year or so.
(The grafana shows 3 loaders had an available memory of 5%)
Nonetheless i can try running with only 4 loaders and see how it goes.

Reply:
fixed and [passed](https://argus.scylladb.com/tests/scylla-cluster-tests/f2ea22f7-6660-4194-bc1d-0015f2adac1d) ok with 4 loaders.